src/js/util/collection.js

```diff
@@ -160,6 +160,21 @@ define(function (require, exports) {
             return !collection.contains(elem);
         });
     };
+    
+    /**
+     * 
```

Comment:
Don't see this being used anywhere... Also, you can just use `Immutable.Set(iterable).toList()` here